client/my-sites/plan-features/style.scss

```diff
@@ -95,6 +95,33 @@ $plan-features-header-banner-height: 20px;
 	margin: 0;
 	font-size: 28px;
 	line-height: 1.5;
+	color: $gray-dark;
+}
+
+.plan-features__price.is-original {
+	color: $gray;
+}
+
+.plan-features__price.is-discounted {
+	color: $alert-green;
+}
+
+.plan-features__price.is-discounted,
+.plan-features__price.is-original {
+	position: relative;
+	align-items: stretch;
+	margin-right: 8px;
+}
+
+.plan-features__price.is-original:before {
+	position: absolute;
+	content: "";
+	left: 0;
+	top: 50%;
+	right: 0;
+	border-top: 2px solid $orange-fire;
+	transform: rotate(-16deg);
```

Comment:
The angle starts looking incorrect at around 4 digits, but I think we'll be ok for our current prices/supported currencies.
<img width="339" alt="screen shot 2016-06-29 at 1 01 39 pm" src="https://cloud.githubusercontent.com/assets/1270189/16466879/b1a574ba-3df9-11e6-8578-ea4ce2972fe3.png">